README.md

```diff
@@ -74,17 +85,17 @@ You bet. All of our custom builders accept "command line style" arguments with t
 
 ##### How is the build process logged?
 
-Each of our custom builders produces a log of its process in the same directory as the first of its targets. Each log is named `sconscript.log` by default, and you can insert custom text between `sconscript` and `.log` by passing it as a string through the builder's `log_ext` keyword argument. It's similar to the way that you specify sources and targets, except that the `log_ext` argument must be a string. It can be important to specify the `log_ext` argument for builders that produce logs in the same directory. The default naming convention allows builders to overwrite one another's logs, so you'll only have a log of the last process to finish. 
+Each of our custom builders produces a log of its process in the same directory as the first of its targets. Each log is named `sconscript.log` by default, and you can insert custom text between `sconscript` and `.log` by passing it as a string through the builder's `log_ext` keyword argument. It's similar to the way that you specify sources and targets, except that the `log_ext` argument must be a string. It can be important to specify the `log_ext` argument for builders that produce logs in the same directory. The default naming convention leads builders to overwrite one another's logs, so you'll only have a log of the last process to finish. 
```

Comment:
I would make the last two sentences more concise.

"It is important to specify the `log_ext` argument for builders that produce logs in the same directory, otherwise the default `sconscript.log` will be overwritten by each builder."